tools/development/gen_mediapipe_libs.sh

```diff
@@ -4,20 +4,24 @@
 if [[ -z "${MEDIAPIPE_HOME}" ]]; then
   echo 'ERROR: Define MEDIAPIPE_HOME First!'
 else
-  # for the external objects
-  for ext_obj in $(find ${MEDIAPIPE_HOME}/bazel-bin/external/ -name '*.o' ); do
-    ext_obj_arr+=($ext_obj)
-  done
-  $(gcc ${ext_obj_arr[@]} -shared -o libmediapipe_external.so -fPIC)
-  $(mv libmediapipe_external.so $1)
+  if [ -d "$1" ]; then
+    # `for the external objects
+    for ext_obj in $(find ${MEDIAPIPE_HOME}/bazel-bin/external/ -name '*.o' ); do
+      ext_obj_arr+=($ext_obj)
```

Comment:
why ($ext_obj)?

```suggestion
      ext_obj_arr+=${ext_obj}
```
or

```suggestion
      ext_obj_arr+=$ext_obj
```

Reply:
since the object files have to be listed like `aa.o bb.o cc.o ...`, the format has been applied. your recommendation generates the list without space between the file name like `aa.obb.occ.o`